src/Types/ClassString.php

```diff
@@ -14,14 +14,13 @@
 namespace phpDocumentor\Reflection\Types;
 
 use phpDocumentor\Reflection\Fqsen;
-use phpDocumentor\Reflection\Type;
 
 /**
  * Value Object representing the type 'string'.
  *
  * @psalm-immutable
  */
-final class ClassString implements Type
+final class ClassString extends String_
```

Comment:
I think we forgot this one when introducing the PseudoType interface, Could you please add that as well here.

Reply:
done